apps/studio/prisma/schema.prisma

```diff
@@ -71,16 +71,21 @@ model Resource {
 
   createdAt DateTime @default(now())
   updatedAt DateTime @default(now()) @updatedAt
+
+  @@unique([siteId, permalink], name: "unique_site_permalink")
```

Comment:
if create page/edit page settings fails, will the user know it is due to the permalink? we should return an appropriate error back